### PR TITLE
added files

### PR DESCRIPTION
**Pull request by faikp**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    content: '';
+    width: 60px;
+    position: absolute;
+    top: 0;
+    left: -10px;
+    bottom: 0;
+    background: linear-gradient(transparent,transparent 13px,black 14px,black 16px,transparent 17px,transparent 30px) repeat-y;
+    box-sizing: border-box;
+    background-size: 30px 30px;
+}
```
Comment (Rehan-Khan-Engineer) [inline style.css]: Could you group similar properties

**Comment by Rehan-Khan-Engineer** [comment]
Its looking Amazing! @faikp

File: style.css (hunk 1 above)
Comment (faikp) [reply style.css]: Ohh , i forgot this one , but I'll do this change soon 👍🏻 @Rehan-Khan-Engineer

File: style.css [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    border: 0;
+
+    font-size: 18px;
+    font-weight: bold;
+    font-family: cursive;
+
+    box-sizing: border-box;
+    
+    resize: none;
+    outline: none;
```
Comment (Abbas4tech) [inline style.css:134]: What does these two property do ??
How it is affecting on webpage @faikp

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

File: style.css (hunk 2 above)
Comment (faikp) [reply style.css:134]: Text area have a resize button  in the ending corner so to hide that we use resize none property when we click on the text area it's have a default outline border to remove those we use Outline none 👍🏻
@Abbas4tech

**Review by Rehan-Khan-Engineer — APPROVED** [review]

**Review by mmusk — APPROVED** [review]
Keep up the good work!

**Review by mujeeb100 — CHANGES_REQUESTED** [review]
there are lots of space are availabe in CSS File.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+    position: absolute;
+    left: 50%;
+}
+.cover{
+    margin: 0 auto;
+
+    min-width: 400px;
+    max-width: 800px;
+    height: 480px;
+
```
Comment (mujeeb100) [inline style.css:27]: Remove the space here

**Review by nasir-k — APPROVED** [review]

**Pull request merged**